pre_commit/commands/run.py

```diff
@@ -255,13 +255,17 @@ def run(runner, store, args, environ=os.environ):
         for repo in repositories(runner.config, store):
             for _, hook in repo.hooks:
                 if (
-                    (not args.hook or hook['id'] == args.hook) and
+                    (not args.hook or hook['id'] == args.hook or (
+                        hook['alias'] and hook['alias'] == args.hook
```

Comment:
the first part of this conditional is unnecessary because we check `not args.hook` above.

I'd write this as (emphasis on indentation here)

```python
            if (
                    (
                        not args.hook or
                        hook['id'] == args.hook or
                        hook['alias'] == args.hook
                    ) and
                    (not hook['stages'] or args.hook_stage in hook['stages'])
            ):
```

Reply:
Fixed.